Cargo.toml

```diff
@@ -24,8 +24,7 @@ backtrace = "0.3.63"
 bytes = "1.1.0"
 lazy_static = "1.4.0"
 libc = { version = "^0.2.112", default-features = false }
-pprof = {version = "^0.5", features = [ "flamegraph", "protobuf" ] }
-prost = "0.8"
```

Comment:
now `pprof` exposes enough of `prost` that the direct dependency is no longer required in `heappy`